tests/python/cli/example_function.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2023 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import cvat_sdk.auto_annotation as cvataa
+import cvat_sdk.models as models
+
+spec = cvataa.DetectionFunctionSpec(
+    labels=[
+        models.PatchedLabelRequest(name="car", id=0),
+    ]
+)
+
+
+def detect(context, image):
```

Comment:
Not everyone is happy with just image bytes as the input. Probably, it would be nice to include the current frame info in the context or function arguments. It also seems reasonable to add a progress callback.

Reply:
I added the frame name to the context.